api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -98,6 +102,10 @@ public void validate(Object target, Errors errors) {
 		
 		// check attributes
 		super.validateAttributes(visit, errors, Context.getVisitService().getAllVisitAttributeTypes());
+
+		validateStartDateTimeToBirthdate(visit, errors);
+
+		validateStartDateTimeToEstimatedBirthdate(visit, errors);
```

Comment:
Not sure if I understand what you're saying, all I'm saying is you can merge the 2 methods into 1 that way you don't have to validate the birthdate that's not estimated twice